### ISSUE TITLE
Custom webpack build docs 404s

### ISSUE THREAD

**Issue by BenjiFischman** [cloudflare/cloudflare-docs#1396]
### Expected Behavior

Here are the webpack versions I am using to orchestrate custom dev, staging, and prod bundling for a react + tailwind thing.

    "webpack-cli": "^4.7.0",
    "webpack-dev-server": "^3.11.2",
    "webpack-merge": "^5.7.3"

For example, I have a root config that merges a dev/staging/prod webpack config with a web pack common config. 
I have no clue if this is tractable with the current documentation.

### Actual Behavior
https://developers.cloudflare.com/workers/cli-wrangler/webpack -> https://developers.cloudflare.com/workers/cli-wrangler/webpack/configuration#build = 404

### Section that requires update
General guidance would be helpful! 
https://developers.cloudflare.com/workers/cli-wrangler/webpack/configuration#build

### Additional information
![image](https://user-images.githubusercontent.com/15211712/119240159-e0285380-bb02-11eb-8949-bc1e12d70e1a.png)

### Example webpack.config.js 

`const { merge, mergeWithCustomize } = require('webpack-merge');
const common = require('./webpack.common.js');
const productionConfig = require('./webpack.prod.js');
const stagingConfig = require('./webpack.staging.js');
const developmentConfig = require('./webpack.dev.js');
`

`module.exports = env =>  {  
    if(env.development || CF_KV.ENVIRONMENT == "development"){
      return merge(common, developmentConfig);
    } else if(env.production || ENVIRONMENT == "production"){
      return merge(common, productionConfig);
    } else if (env.staging || ENVIRONMENT == "staging"){
      return merge(common, stagingConfig);
    } else{
      throw new Error('No matching configuration was found!');
    }
};`

### CL Error on wrangelr dev
![image](https://user-images.githubusercontent.com/15211712/119240378-f7b40c00-bb03-11eb-8a70-022461fcf5c2.png)

**Comment by BenjiFischman** [comment]
For the record, the node webpack stuff works with how I have it configured when I use npm run dev/ build/start, etcetera.

**Comment by BenjiFischman** [comment]
### TL;DR
webpack-merge uses webpack  5 —https://github.com/survivejs/webpack-merge/blob/develop/package.json.

wrangler seems to use  "webpack": "4.33.0". 


### Next Steps?
Any ideas on potential workarounds? 

Meanwhile, I am going to fork and roll back and work within current wrangler support versions.

**Comment by kristianfreeman** [comment]
We've fixed the 404 link here - @BenjiFischman can you open an issue in Wrangler for the webpack dependency issue?
